### DISCUSSION TITLE
Start year of the model

### DISCUSSION THREAD

**Discussion by e-perl-NOAA** [nmfs-stock-synthesis/stock-synthesis#446] [General]
This is a discussion board to help SS3 users with questions about the start year of the model.  Please subscribe to this discussion so that you get notifications about discussion posts.

**Comment by Rick-Methot-NOAA** [comment]
I started by copying relevant sections from the User Manual into this [document ](https://docs.google.com/document/d/160yKgDVe77KD9ca7_7HJgAc-hkLL8HMS1RIrwvxyC2I/edit?usp=sharing) which seems the easiest way to create a draft.  We can move it from there to the vignettes after I get started.

Got it started in outline form.  Let's discuss level of detail and need for pictures and examples before going further.
